src/microbe_stage/systems/MicrobeMovementSystem.cs

```diff
@@ -140,8 +144,19 @@ private static float CalculateRotationSpeed(in Entity entity, ref OrganelleConta
             ref CellProperties cellProperties, ref WorldPosition position,
             ref OrganelleContainer organelles, CompoundBag compounds, float delta)
         {
+            ref var strain = ref entity.Get<StrainAffected>();
+
             if (control.MovementDirection == Vector3.Zero)
             {
+                // Make sure the microbe is not under strain when not moving
+                strain.IsUnderStrain = false;
+
+                // Remove ATP due to strain even if not moving
+                // This is calculated similarily to the regular movement cost for consistency
+                var strainMultiplier = GetStrainMultiplier(ref strain);
```

Comment:
Couldn't this line be straight below the `ref var strain` line as this value seems to be always needed? That way this doesn't need to have the code to calculate this in two places.